### DISCUSSION TITLE
Apple privacy manifest file

### DISCUSSION THREAD

**Discussion by yamade** [react-native-async-storage/async-storage#1059] [Q&A] [answered]
You guys may notice this announcement from Apple.
https://developer.apple.com/support/third-party-SDK-requirements/

I think react-native-async-storage is using fileModificationDate.
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278393

So I'm wondering if you guys have any plan to add a privacy manifest file sometime soon. Thank you!

**Comment by krizzu** [comment, marked as answer]
Hey @yamade, thanks for it, that's a good call. I've created an issue to track this
#1061

**Reply by yamade** [reply]
thanks!
